File: ch34/MvcBasicVB/MvcBasicVB/Controllers/Simple2Controller.vb
Public Class Simple2Controller
    Inherits System.Web.Mvc.Controller

    Sub Hello()
        Response.Write("<h1>Hello World Again!</h1>")
    End Sub

    Sub Goodbye(name As String)
        Response.Write("Goodbye " + HttpUtility.HtmlEncode(name))
    End Sub

End Class

File: ch34/MvcBasicCS/MvcBasicCS/Controllers/Simple2Controller.cs
using System.Web;
using System.Web.Mvc;

namespace MvcBasicCS.Controllers
{
    public class Simple2Controller : Controller
    {
        public void Hello()
        {
            Response.Write("<h1>Hello World Again!</h1>");
        }

        public void Goodbye(string name)
        {
            Response.Write("Goodbye " + HttpUtility.HtmlEncode(name));
        }
    }
}
